### ISSUE TITLE
Tamil Nadu District Maps Request

### ISSUE THREAD

**Issue by justinelliotmeyers** [datameet/maps#46]
I'm updating 2019/ 2020 district boundaries of Tamil Nadu. The last split am working on is Kancheepuram/ Chengalpattu. I found a map here: https://cdn.s3waas.gov.in/s31543843a4723ed2ab08e18053ae6dc5b/uploads/2018/06/2018060830.pdf

and a document here: https://cms.tn.gov.in/sites/default/files/go/revenue_e_428_2019.pdf

But was wondering if anyone knows where to locate a better map of the actual district boundaries for 2019/ 2020. Also, Chennai absorbed several districts so I would like to properly account for that. Any help would be greatly appreciated. @thejeshgn @devdattaT @planemad @geohacker @NishaT

**Comment by planemad** [comment]
The updated extent of Chennai district should match the new city boundaries as mentioned in https://en.wikipedia.org/wiki/Chennai_district

City boundary: https://github.com/datameet/Municipal_Spatial_Data/blob/master/Chennai/Wards.geojson

**Comment by justinelliotmeyers** [comment]
@planemad thanks for the update. I looked for several hours for an updated map, and had no luck. I sent out several emails to different government offices, but haven't heard anything. For Now, I modified the surrounding areas in my working dataset to reflect what I could based on the limited information I had. Final updates to geometry are in an issue here: https://github.com/justinelliotmeyers/India_Official_Boundaries_2019/issues/4
